storybook/src/components/Heading/Heading.stories.tsx

```diff
@@ -70,3 +78,21 @@ export const InverseColour: Story = {
     level: 2,
   },
 }
+
+export const WithIcon: Story = {
+  args: {
+    children: 'Heading text',
+    level: 4,
+  },
+  argTypes: {
+    level: {
+      table: { disable: true },
+    },
+  },
```

Comment:
Why hide `level` here?

Reply:
Because I hadn’t taken the effort to fix all possible variations of this example. Now I have.